src/main/java/com/emc/pravega/perf/PravegaPerfTest.java

```diff
@@ -400,76 +602,70 @@ public void run() {
                 return null;
 
           };
-        }
-    }
+      }
+}
 
-    /**
-     * A Sensor reader class that reads the temperative data
-     */
-    private static class ReaderWorker implements Runnable {
-       @Setter
-        public static AtomicInteger totalEvents;
-        private EventStreamReader<String> reader;
-        String readerId;
+class ReaderWorker implements Callable<Void> {
+       public static AtomicInteger totalEvents;
```

Comment:
I think this should be non-static.

Reply:
no, this is AtomicInteger  static used by multiple  readers